kebechet/utils.py

```diff
@@ -43,7 +45,7 @@ def cwd(path: str):
 
 
 @contextmanager
-def cloned_repo(service_url: str, slug: str, **clone_kwargs):
+def cloned_repo(manager: Any, service_url: str, slug: str, **clone_kwargs):
```

Comment:
Would it make sense to reduce number of parameters supplied? `service_url`, `slug` can be taked directly from `manager` (which is of type manager base).

Reply:
Yes, I will get those from the manager class, rather than passing it as arguments :+1: Thanks